src/alert-manager/config/alert-manager.md

```diff
@@ -50,4 +53,5 @@ alert-manager:
     host: master_ip
     url: "http://master_ip:9093"
     use-pylon: False
+    repeat-interval: 4h
```

Comment:
we can extend it to 24h, and depends on resolved mails.

Reply:
yes, I provide this as a config, user who want to extend can extend when deploying.